apps/www/pages/klimalabor.tsx

```diff
@@ -0,0 +1,20 @@
+import Frame from '../components/Frame'
+import { Button, colors } from '@project-r/styleguide'
+
+const Klimalabor = () => {
+  const meta = {
+    title: 'Klimalabor Landing Page',
+    description: 'Langing Page',
+  }
+  return (
+    <Frame meta={meta} isClimate={true}>
+      <p>
+        Und hier kommt <span>Und hier dann ein gelber Balken dahiner</span>
+        viel Inhalt
+      </p>
+      <Button primary>Das ist ein Button</Button>
+    </Frame>
+  )
+}
+
+export default Klimalabor
```

Comment:
Get rid of this page, or else with the next FE-deployment, people will be able to see this under `www.republik.ch/klimalabor`.